addons/rose/tests/integration/components/rose/form/fieldset-test.js

```diff
@@ -0,0 +1,79 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | rose/form/fieldset', function (hooks) {
+  setupRenderingTest(hooks);
+
+  // Global values for mocking
+  this.mockTitle = 'Title';
+  this.mockDescription = 'Description';
+  this.mockBody = 'Rest of the form';
+
+  // Wrap the rendering in a function to avoid repeat code
+  async function renderComponent(noDescription) {
```

Comment:
@randallmorey Here I changed the test. Previously I wrapped the rendering function within `beforeEach()` but since we need to render without description too this is the solution I came with.